src/psyclone/tests/test_files/dynamo0p3/infrastructure/mesh_mod.F90

```diff
@@ -272,27 +272,32 @@ module mesh_mod
 
     procedure, public :: get_last_inner_cell
     procedure, public :: get_last_inner_cell_per_colour
+    procedure, public :: get_last_inner_cell_all_colours
 
     procedure, public :: get_num_cells_edge
 
     procedure, public :: get_last_edge_cell
     procedure, public :: get_last_edge_cell_per_colour
+    procedure, public :: get_last_edge_cell_all_colours
 
     procedure, public :: get_halo_depth
     procedure, public :: get_num_cells_halo
 
     procedure, public :: get_last_halo_cell_any
     procedure, public :: get_last_halo_cell_deepest
     generic           :: get_last_halo_cell => &
-                            get_last_halo_cell_any, &
-                            get_last_halo_cell_deepest
+                           get_last_halo_cell_any, &
```

Comment:
The removal of a space for these two lines makes the indent different to further down (which is three) Is there a reason for this?

Reply:
Who knows? I've put the space back.